.github/workflows/release.yml

```
@@ -61,4 +61,21 @@ jobs:
           git add .
           git commit -m "chore(): update package lock files"
           git push
+      # Purge the JSDeliver CDN cache so
+      # component playgrounds always load
+      # the latest version of Ionic.
+      - name: Purge JSDelivr Cache
+        run: | 
+          curl -X POST \
+            http://purge.jsdelivr.net/ \
```

```suggestion
            https://purge.jsdelivr.net/ \
```

This should probably be over https.